torch/csrc/jit/codegen/cuda/compute_at.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <c10/util/Exception.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+
+struct TensorView;
+
+struct ComputeAt {
```

Comment:
can we make this a `class` (and order the members from public, then private methods and finally private data) ?

or even better - split the configuration from the action and have a simple interface like:

```
struct ComputeAtContext {
  TensorView* producer = nullptr;
  TensorView* consumer = nullptr;
  unsigned int consumer_position = 0;
};

void runComputeAt(const ComputeAtContext& context);
```

This way everything irrelevant to the users are really hidden and we have a simple & small interface.